### PR TITLE
:tada: alakarte

### PR DESCRIPTION
**Pull request by guillaumecrespel**

### CODE REVIEW

**Review by fabienjuif — COMMENTED** [review]
I used a :tada: for other rebranding.

`:tada: alakarte` (to respect `<emoji> <message in lowercase>`)

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
-# Trampss TodoMVC Example
-[![CircleCI](https://circleci.com/gh/Trampss/todomvc.svg?style=svg)](https://circleci.com/gh/Trampss/todomvc) [![Coverage Status](https://coveralls.io/repos/github/Trampss/todomvc/badge.svg?branch=master)](https://coveralls.io/github/Trampss/todomvc?branch=master)
+# Our best way to make TodoMVC Example
+[![CircleCI](https://circleci.com/gh/Trampss/todomvc.svg?style=svg)](https://circleci.com/gh/Alakarte/todomvc) [![Coverage Status](https://coveralls.io/repos/github/Trampss/todomvc/badge.svg?branch=master)](https://coveralls.io/github/Alakarte/todomvc?branch=master)
```
Comment (fabienjuif) [inline README.md]: alakarteio

File: src/components/info/info.jsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import React from 'react'
 export default () => (
   <div className="info">
     <p>Double-click to edit a todo</p>
-    <p>Created by <a href="https://github.com/Trampss"><b>Trampss - 🐲</b></a></p>
+    <p>Created by <a href="https://github.com/Alakarteio"><b>Alakarte - 🐲</b></a></p>
```
Comment (fabienjuif) [inline src/components/info/info.jsx]: `alakarteio`

File: README.md [hunk 3]
```diff
@@ -1,5 +1,8 @@
-# Trampss TodoMVC Example
-[![CircleCI](https://circleci.com/gh/Trampss/todomvc.svg?style=svg)](https://circleci.com/gh/Trampss/todomvc) [![Coverage Status](https://coveralls.io/repos/github/Trampss/todomvc/badge.svg?branch=master)](https://coveralls.io/github/Trampss/todomvc?branch=master)
+# Our best way to make TodoMVC Example
```
Comment (fabienjuif) [inline README.md]: Nope

File: src/components/info/info.jsx (hunk 2 above)
Comment (fabienjuif) [reply src/components/info/info.jsx]: alakarteio

**Review by fabienjuif — CHANGES_REQUESTED** [review]
see comments

File: README.md (hunk 3 above)
Comment (guillaumecrespel) [reply README.md]: "Todo MVC with ReactJS + redux-saga + Redux" ?

Comment (fabienjuif) [reply README.md]: If you are proud and want to show it, maybe something like `with Alakarte stack`.
And we could make a link to our talk on the title later ?

**Review by fabienjuif — APPROVED** [review]

**Pull request merged**